src/components/Grid/Grid.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import settings from '../../globals/js/settings';
+
+const { prefix } = settings;
+
+/** Grid layout */
+const Grid = React.forwardRef(
+  (
+    { gutter, children, grow, justify, align, columns, className, ...other },
+    ref
+  ) => {
+    const wrapperClasses = classnames(`${prefix}--grid`, {
+      [`${prefix}--grid--spacing-xl`]: gutter === 'xl',
+      [`${prefix}--grid--spacing-lg`]: gutter === 'lg',
+      [`${prefix}--grid--spacing-md`]: gutter === 'md',
+      [`${prefix}--grid--spacing-sm`]: gutter === 'sm',
+      [`${prefix}--grid--spacing-xs`]: gutter === 'xs',
+      [`${prefix}--grid--spacing-none`]: gutter === 'none',
+      [`${prefix}--grid--grow`]: grow,
+      [`${className}`]: className,
+    });
+    return (
+      <div
+        ref={ref}
+        className={wrapperClasses}
+        {...other}
+        style={{ justifyContent: justify, alignItems: align }}>
```

Comment:
Can the user set different values for `justifyContent` and `alignItems` on mobile?